pages/index.tsx

```diff
@@ -22,6 +24,11 @@ import AllPositions from "../features/all-positions/AllPositions";
 import Weth from "../features/weth/Weth";
 import Yield from "../features/yield/Yield";
 
+import GitHubIcon from "@material-ui/icons/GitHub";
+import DiscordIcon from "../public/discord.svg";
+import TwitterIcon from "../public/twitter.svg";
+import VercelIcon from "../public/vercel.svg";
```

Comment:
Also, I'm pretty sure we are not using the vercal Icon?

Reply:
Yeah we're not my bad